index.html

```diff
@@ -1955,8 +1955,8 @@
             Interface <dfn>PresentationConnectionAvailableEvent</dfn>
           </h4>
           <pre class="idl">
-            [SecureContext,
-             Constructor(DOMString type, PresentationConnectionAvailableEventInit eventInitDict)]
+            [Constructor(DOMString type, PresentationConnectionAvailableEventInit eventInitDict),
+            SecureContext, Exposed=Window]
```

Comment:
Nit: Add a space to align indent with previous line.

Reply:
Fixed.